src/services/source-statuses.service.ts

```diff
@@ -1,7 +1,67 @@
+import { NotUniqueError } from '../errors/not-unique.error';
+import { NotFoundError } from '../errors/not-found.error';
+import { RemovalRestrictedError } from '../errors/removal-restricted.error';
 import { SourceStatus } from '../models/source-status.model';
 
 export class SourceStatusesService {
+  /** GET all source statuses */
   static async getAll() {
     return await SourceStatus.find();
   }
+
+  /** GET one source status */
```

Comment:
Comments describing HTTP methods should not have any relation to methods in services.